### PR TITLE
images: Update rhel-8-3 to z-stream

### PR DESCRIPTION
**Pull request by marusak**
* [ ] FAIL: image-refresh rhel-8-3

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 1-ci-srv-01.
Log: https://logs.cockpit-project.org/logs/image-refresh-1339-20201026-120404/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-1339-20201026-120404/

**Comment by cockpituous** [comment]
image-refresh in progress on 4-ci-srv-03.
Log: https://logs.cockpit-project.org/logs/image-refresh-1339-20201026-115858/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-1339-20201026-115858/

**Comment by cockpituous** [comment]
image-refresh in progress on 1-ci-srv-05.
Log: https://logs.cockpit-project.org/logs/image-refresh-1339-20201026-121111/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-1339-20201026-121111/

**Comment by cockpituous** [comment]
image-refresh in progress on 2-cockpit-10.
Log: https://logs.cockpit-project.org/logs/image-refresh-1339-20201026-135454/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-1339-20201026-135454/

**Comment by marusak** [comment]
`ERROR    Couldn't find kernel for install tree.` oh common

**Comment by cockpituous** [comment]
image-refresh in progress on 3-ci-srv-02.
Log: https://logs.cockpit-project.org/logs/image-refresh-1339-20201027-112525/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-1339-20201027-112525/

**Comment by cockpituous** [comment]
image-refresh in progress on 1-cockpit-10.
Log: https://logs.cockpit-project.org/logs/image-refresh-1339-20201029-112020/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-1339-20201029-112020/

**Comment by cockpituous** [comment]
image-refresh in progress on 1-cockpit-11.
Log: https://logs.cockpit-project.org/logs/image-refresh-1339-20201102-123939/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-1339-20201102-123939/

**Comment by cockpituous** [comment]
image-refresh in progress on 1-cockpit-7.
Log: https://logs.cockpit-project.org/logs/image-refresh-1339-20201201-100202/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-1339-20201201-100202/

File: images/scripts/rhel-8-3.bootstrap [hunk 1]
```diff
@@ -3,4 +3,4 @@
 set -ex
 
 BASE=$(dirname $0)
-$BASE/virt-install-fedora "$1" x86_64 "http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.3/compose/BaseOS/x86_64/os/"
+$BASE/virt-install-fedora "$1" x86_64 "http://download.devel.redhat.com/rhel-8/nightly/updates/RHEL-8/latest-RHEL-8.3.1/compose/BaseOS/x86_64/os/"
```
Comment (mvollmer) [inline images/scripts/rhel-8-3.bootstrap:6]: The inclusion of `updates` is suspicious here.  I wouldn't be surprised if one can not bootstrap from any `update` repository... but then again `/rhel-8/nightly/updates/RHEL-8/latest-RHEL-8.0.0/compose/BaseOS/x86_64/os/` does have isolinux/... I will never understand how RHEL is made...

Comment (mvollmer) [reply images/scripts/rhel-8-3.bootstrap:6]: See #1425 for a try with just latest-RHEL-8.3.1.

Comment (martinpitt) [reply images/scripts/rhel-8-3.bootstrap:6]: That's a good point -- let's keep the release repo here, and only apply updates in the setup script (as it already does).

Comment (mvollmer) [reply images/scripts/rhel-8-3.bootstrap:6]: Why would a nightly build already have updates?

Comment (martinpitt) [reply images/scripts/rhel-8-3.bootstrap:6]: Ah right, I somehow thought .1 was already out, but of course the point of all this is exactly to test the *pre*releases.

**Comment by marusak** [comment]
Obsoleted by #1425

**Pull request closed without merging**